README.md

```diff
@@ -119,6 +119,12 @@ The above command sets the Galaxy persistent volume to 50GB.
 
 Setting Galaxy configuration file values requires the key name to be escaped:
 
+```console
+helm install --set-file "configs.galaxy\.yml.brand"="Hello World
```

Comment:
Missing a closing `"`?

Reply:
Thanks for picking that up. Have fixed in new push.